presto-main/src/main/java/com/facebook/presto/operator/project/PageProcessor.java

```diff
@@ -103,6 +105,17 @@ public PageProcessorOutput process(ConnectorSession session, Page page)
         return projections;
     }
 
+    private static long calculatePageRetainedSizeWithoutLoading(Page page)
+    {
+        long retainedSizeInBytes = 0;
+        for (Block block : page.getBlocks()) {
+            if (!(block instanceof LazyBlock)) {
+                retainedSizeInBytes += block.getRetainedSizeInBytes();
```

Comment:
I don't know the context, but what would happen if `LazyBlock.getRetainedSizeInBytes` returned ~0 if block is not loaded?